festim/helpers_discontinuity.py

```diff
@@ -0,0 +1,223 @@
+from mpi4py import MPI
+import dolfinx
+import dolfinx.fem.petsc
+import ufl
+import numpy as np
+from petsc4py import PETSc
+import basix
+
+
+class NewtonSolver:
```

Comment:
Eventually we will replace this by the solver provided by https://github.com/scientificcomputing/scifem

Reply:
Done